numpy/ma/core.py

```diff
@@ -25,7 +25,6 @@
 import sys
 import operator
 import warnings
-import textwrap
```

Comment:
Unless the cost is really high, I think textwrap aids readability too much to be worth removing. If the cost really is that high, then can we vendor the dedent function?

Reply:
Running benchmarks on my laptop is making it really hot. At first glance, it looks like removing textwrap shaves about 8ms / 80 ms. 

I really disagree with the claim that textwrap.deindent improves readability.

Deindent seems more like it should be used where a user inputs unpredictable text, whereas here, we can just press shift tab or add a new line to align things the way we want.

Reply:
Looks like the cost stems from an `re` import. Turns out matplotlib vendors dedent for performance reasons, I'd be inclined to follow their lead.

Reply:
re is a really hard import to get rid of.

I tried as part of this. Re is used a little everywhere to parse strings (often it really isn't necessary) but I did find a few precompiled parsers which were too complex to get rid of.

Reply:
Ok, i included your thoughts above in the first post. I'm not the one doing regular maintenance, so you all can make the final decision taking everything into account. Either fix is likely going to be ok.